src/pkg/api/start.go

```diff
@@ -258,36 +258,40 @@ func fileServer(r chi.Router, root http.FileSystem) error {
 }
 
 func Serve(r *chi.Mux, localCert []byte, localKey []byte, inCluster bool) error {
-	//nolint:gosec,govet
 	if inCluster {
 		slog.Info("Starting server in in-cluster mode on 0.0.0.0:8080")
-
+		//nolint:gosec,govet
 		if err := http.ListenAndServe("0.0.0.0:8080", r); err != nil {
 			message.WarnErrf(err, "server failed to start: %s", err.Error())
 			return err
 		}
-	} else {
-		slog.Info("Starting server in local mode on 127.0.0.1:8443")
-		// create tls config from embedded cert and key
-		cert, err := tls.X509KeyPair(localCert, localKey)
-		if err != nil {
-			log.Fatalf("Failed to load embedded certificate: %v", err)
-		}
-		tlsConfig := &tls.Config{
-			Certificates: []tls.Certificate{cert},
-		}
+		return nil
+	}
+	slog.Info("Starting server in local mode on 127.0.0.1:8443")
+	if inAirgap {
+		err := serveAirgap(r)
+		return err
+	}
 
-		// Create a server with TLS config
-		server := &http.Server{
-			Addr:      "127.0.0.1:8443",
-			Handler:   r,
-			TLSConfig: tlsConfig,
-		}
+	// connected to internet, create tls config from embedded cert and key
+	cert, err := tls.X509KeyPair(localCert, localKey)
+	if err != nil {
+		log.Fatalf("Failed to load embedded certificate: %v", err)
```

Comment:
`log.Fatalf()` logs a message then invokes `os.Exit(1)`. We invoke `api.Serve()` from `main.go` ~line 33 where, where if `Serve()` returns an error we `os.Exit(1)` so we can simply log the error here and return it. This keeps up with using `slog` while passing control back to `main.go` 

```suggestion
		slog.Error("failed to load embedded certificate", "error", err)
		return err
```